File: XWeather/Shared/Domain/Wu/Hourly.cs
using System.Collections.Generic;

namespace XWeather.Domain
{
	public class Hourly : WuObject
	{
		public override string WuKey => "hourly";

		public List<HourlyForecast> hourly_forecast { get; set; }
	}

	public class HourlyForecast
	{
		public FCTTIME FCTTIME { get; set; }
		public Measurement temp { get; set; }
		public Measurement dewpoint { get; set; }
		public string condition { get; set; }
		public string icon { get; set; }
		public string icon_url { get; set; }
		public string fctcode { get; set; }
		public string sky { get; set; }
		public Measurement wspd { get; set; }
		public Measurement wdir { get; set; }
		public string wx { get; set; }
		public string uvi { get; set; }
		public string humidity { get; set; }
		public Measurement windchill { get; set; }
		public Measurement heatindex { get; set; }
		public Measurement feelslike { get; set; }
		public Measurement qpf { get; set; }
		public Measurement snow { get; set; }
		public double pop { get; set; }
		public Measurement mslp { get; set; }
	}
}

File: XWeather-Swift/XWeather/Domain/Hourly.swift
//
//  Hourly.swift
//  XWeather
//
//  Created by Colby Williams on 6/11/17.
//  Copyright © 2017 Colby Williams. All rights reserved.
//

import Foundation

public class Hourly : WuObject
{
	public override var WuKey: String { return "hourly" }
	
	public var hourly_forecast: [HourlyForecast] = []
}

public class HourlyForecast
{
	public var FCTTIME: FCTTIME?
	public var temp: Measurement?
	public var dewpoint: Measurement?
	public var condition: String?
	public var icon: String?
	public var icon_url: String?
	public var fctcode: String?
	public var sky: String?
	public var wspd: Measurement?
	public var wdir: Measurement?
	public var wx: String?
	public var uvi: String?
	public var humidity: String?
	public var windchill: Measurement?
	public var heatindex: Measurement?
	public var feelslike: Measurement?
	public var qpf: Measurement?
	public var snow: Measurement?
	public var pop: Double?
	public var mslp: Measurement?
}
